packages/testing/test/database.test.ts

```diff
@@ -87,13 +87,16 @@ describe('Testing Module Tests', function() {
   });
 
   it('loadDatabaseRules() throws if no databaseName or rules', async function() {
+    //@ts-ignore This test is checking an invalid param.
     await expect(firebase.loadDatabaseRules.bind(null, {})).to.throw(
```

Comment:
`(firebase as any).loadDatabaseRules` and use eslint suppression instead.

Reply:
Done for 3 lines.